### PR TITLE
At long last, Fire Extingushers

### PR DESCRIPTION
**Pull request by Eryneril**

### CODE REVIEW

**Comment by bleehu** [comment]
@Eryneril, don't forget to set your reviewers as assignees and leave a helpful message as a commit message. It'll help @Turtlelord26 when he's doing the changelog to know what this commit did. 

This PR would handle Issue #560

File: Items/07_Items.txt [hunk 1]
```diff
@@ -20,6 +20,18 @@ Fishing Pole ($120):
 sharp hook. Bait not included. An expert angler could potentially cast this out
 to 6 meters.  
 
+Fire Extinguisher ($115):
+	A handheld cylindrical trigger-based vessel with two mag slots, 
+one for a pressurized fire suppressing agent and one for a pressurized 
+propellant. Can be used to put out sources of fire and has a range of 
+4m. Requires both hands to operate. Not recommended for use on anything 
+other than a fire.
```
Comment (bleehu) [inline Items/07_Items.txt]: Can we add a note that the extinguisher comes with a set of mags?

**Review by bleehu — APPROVED** [review]
one nonblocking request. Otherwise, it looks good!

**Comment by Turtlelord26** [comment]
nuuu not an "Update [filename]" commit title 😦

**Pull request merged**